contracts/Tokens/VTokens/VToken.sol

```diff
@@ -351,6 +344,168 @@ abstract contract VToken is VTokenInterface, Exponential, TokenErrorReporter {
         protocolShareReserve = protcolShareReserve_;
     }
 
+    /**
+     * @notice Transfers the underlying asset to the specified address.
+     * @dev Can only be called by the Comptroller contract. This function performs the actual transfer of the underlying
+     *      asset by calling the `doTransferOut` internal function.
+     *      - The caller must be the Comptroller contract.
+     *      - If the `to` address is not the protocol share reserve, the flashLoanAmount is incremented by the amount transferred out.
+     * @param to The address to which the underlying asset is to be transferred.
+     * @param amount The amount of the underlying asset to transfer.
+     * @return balanceBeforeRepayFlashloan Cash balance after transfer out for comparison in flash loan verification.
+     * @custom:error InvalidComptroller is thrown if the caller is not the Comptroller.
+     * @custom:event Emits TransferOutUnderlying event on successful transfer of amount to receiver
+     */
+
+    function transferOutUnderlying(
+        address payable to,
+        uint256 amount
+    ) external nonReentrant returns (uint256 balanceBeforeRepayFlashloan) {
+        if (msg.sender != address(comptroller)) {
+            revert InvalidComptroller();
+        }
+
+        if (to != protocolShareReserve) {
+            flashLoanAmount += amount;
+        }
+        doTransferOut(to, amount);
+
+        balanceBeforeRepayFlashloan = getCashPrior();
+        emit TransferOutUnderlying(underlying, to, amount);
+    }
+
+    /**
+     * @notice Transfers the underlying asset from the specified address.
+     * @dev Can only be called by the Comptroller contract. This function performs the actual transfer of the underlying
+     *      asset by calling the `doTransferIn` internal function.
+     *      - The caller must be the Comptroller contract.
+     * @param from The address from which the underlying asset is to be transferred.
+     * @param amountRepayed The amount of the underlying asset to transfer.
+     * @custom:error InvalidComptroller is thrown if the caller is not the Comptroller.
+     * @custom:event Emits TransferInUnderlyingAndVerify event on successful transfer of amount from the receiver to the vToken
+     */
+    function transferInUnderlyingAndVerify(address payable from, uint256 amountRepayed) external nonReentrant {
```

Comment:
Why the suffix `Verify`? I would add `flashLoan` as a prefix. I would also add `flashLoan` as a prefix to the function `transferOutUnderlying`

Reply:
https://github.com/VenusProtocol/venus-protocol/pull/545/commits/8ed2d89f60173c908de03db73c5d99d492c939ac